jsapp/js/account/organizations/membersQuery.ts

```diff
@@ -1,10 +1,21 @@
 import {keepPreviousData, useQuery} from '@tanstack/react-query';
 import {endpoints} from 'js/api.endpoints';
 import type {PaginatedResponse} from 'js/dataInterface';
-import {fetchGet} from 'js/api';
+import {fetchGet, fetchPatch, fetchDelete} from 'js/api';
 import {QueryKeys} from 'js/query/queryKeys';
 import {useOrganizationQuery} from '../stripe.api';
 
+/**
+ * Note that it's only possible to update the role via API to either `admin` or
+ * `member`.
+ */
+export enum OrganizationMemberRole {
```

Comment:
Note to self @magicznyleszek: there is already `OrganizationUserRole` with same-ish values